.github/workflows/deploy-enterprise.yml

```
@@ -42,6 +42,21 @@ jobs:
             ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
             distribution_url: ''
 
+          - environment: hyp3-nisar-jpl
+            domain: hyp3-nisar-jpl.asf.alaska.edu
+            template_bucket: cf-templates-gdeyr9hh8rzs-us-west-2
+            image_tag: latest
+            product_lifetime_in_days: 180
+            quota: 0
+            job_files: job_spec/INSAR_ISCE.yml job_spec/INSAR_ISCE_TEST.yml
+            instance_types: c5d.xlarge
+            default_max_vcpus: 1600
+            expanded_max_vcpus: 1600
+            required_surplus: 0
+            security_environment: JPL-public
```

```suggestion
            security_environment: JPL
```
I forgot to mention this bit -- we need to use the `JPL` security environment until the public bucket request is done. After we can switch it to `JPL-public` which will make the HyP3 content bucket public.